src/chromium/adding-third-party-crates.md

```diff
@@ -0,0 +1,25 @@
+# Adding third party crates
+
+Rust libraries are called "crates" and are found at [crates.io][0]. It's *very
+easy* for Rust crates to depend upon one another. So they do!
+
+| Property | C++ library | Rust crate |
+| --- | --- | --- |
+| Build system | Lots | Consistent - `Cargo.toml` |
+| Typical library size | Large | Small |
```

Comment:
nitty nit: "Large" is very true for Chromium C++ targets.  But it's not necessarily true for *all* C++ targets.

Not sure if an action is needed here.  I guess you could qualify "C++ library" as "Chromium C++ library", but maybe this just adds noise that is unnecessary for most readers/attendees.

Reply:
Tweaked